src/main/java/controller/MainController.java

```diff
@@ -66,20 +70,46 @@ public void hideStartMessage() {
      * Show lives in the top bar.
      *
      * @param lives The number of lives.
+     * @param playerNumber The number of the player (used for multiplayer).
      */
-    public void showLives(int lives) {
+    public void showLives(int lives, int playerNumber) {
+
         livesText.setVisible(true);
-        livesText.setText(String.format("Lives: %d", lives));
+
+        if (playerNumber == 1) {
+            livesTextPlayer1.setVisible(true);
+
+            if (StartController.getLimitOfPlayers() == 1) {
+                livesTextPlayer1.setText(String.format("%d", lives));
+            } else {
+                livesTextPlayer1.setText(String.format("%d (Player 1)", lives));
+            }
+        }
+
+        if (playerNumber == 2) {
```

Comment:
Why not use ```else if```? This is more efficient here, because ```playerNumber``` can never be 1 and 2 at the same time.